### PR TITLE
Setup Typography

### LINKED ISSUE #32: Setup Typography
Checklist:

- [ ]  Setup tailwind to use the typography as described in the image. The following custom tailwind classes should be added to our tailwind config file:
    - [ ]  `text-h1` → Heading 1
    - [ ]  `text-h2` → Heading 2
    - [ ]  `text-subtitle` → Subtitle
    - [ ]  `text-normal` → paragraph, body text, etc…
    - [ ]  `text-dropdown` → dropdown text
- [ ]  Set a global to use the typography as described in the image for the following tags (add bold and size as well, we should stick to HTML semantics as much as possible):
    - [ ]  `h1` → Heading 1
    - [ ]  `h2` → Heading 2
    - [ ]  `h3` → Subtitle
    - [ ]  `p` `span` `label` or any other text that is not in a heading tag

### PR DESCRIPTION
**Pull request by ABCodez**
Fixes #32:
image

- setting custom tailwind classes wasnt needed as the tags are already set in index.css for the entire app.
- font sizes MAY need to be changed in the future due to potential sizing issues.

### CODE REVIEW

File: tailwind.config.js
Comment (juancwu) [inline tailwind.config.js:1]: Can you rename this file to `.mjs` and use the es module import and export syntax? I missed this when reviewing your last PR but our package.json is working in module mode.

**Review by juancwu — CHANGES_REQUESTED** [review]

File: tailwind.config.js
Comment (ABCodez) [reply tailwind.config.js:1]: yup, will get it done

**Review by juancwu — APPROVED** [review]
lgtm

File: index.html [hunk 1]
```diff
@@ -9,6 +9,11 @@
 			href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400;500;600;700;800;900&display=swap"
 			rel="stylesheet"
 		/>
+
+		<link
```
Comment (aidantrabs) [inline index.html:13]: Let's use imports for fonts in `index.css` instead of links in the `index.html`

- Change this for both Raleway and Poppins

**Review by aidantrabs — CHANGES_REQUESTED** [review]
See comment

File: index.html (hunk 1 above)
Comment (ABCodez) [reply index.html]: Resolved in most recent PR

**Pull request closed without merging**